kbase-extension/scss/partials/_bootstrapHelper.scss

```diff
@@ -5,6 +5,10 @@
     Customize bootstrap styles
 */
 
+.text-silver {
```

Comment:
bootstrap has a load of `text-(danger|success|primary|warning)` colours -- this adds silver to that set.